api/ibiconfig/ibiconfig.go

```diff
@@ -79,6 +79,9 @@ type IBIPrepareConfig struct {
 	// Provide the device id like /dev/by-id/ata-xxxxx
 	InstallationDisk string `json:"installationDisk"`
 
+	// ReleaseRegistry is the container image registry that hosts the OpenShift release-image content.
+	ReleaseRegistry string `json:"releaseRegistry,omitempty"`
```

Comment:
Please mention that this registry is used during pre-caching (while performing the preparation/installation step) and can be different than the registry to be used at the edge site deployment

Reply:
Great point, I've updated the doc to reflect this.